release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -165,6 +165,35 @@ For more information, see xref:../installing/installing_aws/installing-restricte
 [id="ocp-4-11-web-console"]
 === Web console
 
+[id="ocp-4-11-dynamic-plug-in-updates"]
+==== Dynamic plug-in updates
+
+With this update, you can now enable and disable plug-ins in the *Administration* -> *Cluster Settings* page of the web console. You can also add different extension types to customize your plug-in.
+
+For more information, see xref:../web_console/dynamic-plug-ins.adoc#dynamic-plug-ins_web_console[Dynamic plug-ins].
+
+[id="ocp-4-11-dark-mode"]
+==== Support for dark mode theme
+
+The {product-title} web console now supports the dark mode theme. On the *User Preferences* page, select your preferred mode to view the web console in.
+
+[id="ocp-globally-installed-operators-display-operand-instances"]
+==== Display operand instances for all managed namespaces on the *Installed Operator* page
+
+With this update, the default view for globally installed Operators in the *All namespaces* and *Operand* tabs of the *Installed Operator* -> *Details* page will show all Operators across all namespaces. You are still able to view only the instances in the selected namespace within the project selector. When viewing the operand instances, a new radio switching control allows all operand instances from either all namespaces or only the current namespace to be seen.
+
+[id="ocp-4-11-conditional-updates"]
+==== Conditional updates
+
+With this update, if conditional updates are available, you can enable `Include supported but not recommended versions` in the `Select new version` dropdown of the *Update cluster* modal to populate the dropdown list with conditional updates. If a `Supported but not recommended` version is selected, an alert will appear below the dropdown displaying potential issues with the version.
+
+[id="ocp-4-11-pod-disruption-budgets"]
+==== Pod disruption budgets (PDBs)
+
+This update provides support for pod disruption budgets (PDBs) to the {product-title} web console. From *Workloads* -> *PodDisruptionBudget*, you can create PDBs for pod resources. Alternatively, pod disruption budgets can be created from pod controller resources list and detail pages. For example, from *Deployments* -> click `Add PodDisruptionBudget`. You can also select `PodDisruptionBudget` and change the availability requirement to `MinAvailable` or `maxAvailable` to specify the pods running.
```

Comment:
nav menu is in plural forms `PodDisruptionBudgets`  so it should be `Workloads -> PodDisruptionBudgets`

Reply:
@opayne1  what does this sentence `You can also select `PodDisruptionBudget` and change the availability` mean?  where did use select `PodDisruptionBudget`?

Reply:
@yapei I have updated this to reflect your comments. Thank you!

Reply:
@opayne1  thanks for your updates, so minor comments based on latest changes
```
For example, from Workloads → Deployments click Add PodDisruptionBudget. You can also change the availability requirements for the maximum number of unavailable Pods and the maximum number of surge Pods during the rolling deployment.
```
`the maximum number of surge Pods` looks quite different from `minAvailable`

Reply:
@yapei Oh I see! I had a cluster up and went into *Deployments* based on the sentence before. I see now that if you are in *PodDisruptionBudgets* and you click `PodDisruptionBudget` you can select a `maxUnavailable` and `minAvailable` from the availability requirement list and set the value of pods running. I think I will move this sentence up to avoid confusion.

Reply:
Updated!